corehq/apps/enterprise/metrics.py

```diff
@@ -0,0 +1 @@
+ENTERPRISE_API_ACCESS = 'enterprise_api_access'
```

Comment:
Nit: What about rename the file to `events.py` since those values are passed as `event_name` parameter?

Reply:
Addressed in b2b68907bdf4c9ca1e119328d08ccf79639f2c45
I opted to name is 'metric_events' instead, as I think 'event' can lead to confusion with event-driven programming